app/src/main/java/space/taran/arkretouch/presentation/edit/rotate/RotateGrid.kt

```diff
@@ -167,9 +164,11 @@ class RotateGrid {
 
     fun get() = rect
 
-    fun getBitmap() = bitmap
+    fun getBitmap() = bitmap.asAndroidBitmap()
 
-    fun draw(canvas: Canvas) {
+    fun draw(canvas: Canvas, angle: Float = 0f) {
+        // canvas.rotate(angle, pivot.x, pivot.y)
```

Comment:
Commented-out code should not be merged into `main`.

Reply:
We don't really need ```RotateGrid``` now, so I will remove it unless it is need later